### ISSUE TITLE
Add Larvae and Eggs events to SmartDots.

### ISSUE THREAD

**Issue by cmspinto** [ices-eg/SmartDots#191]
Talked with Matthias and he suggested that events of Larvae and Eggs could also be added to SmartDots. These events are not exactly like Maturity but the flow is similar. 
Matthias will sent more details during this week.

**Comment by cmspinto** [comment]
Example of fish eggs annotations

![Fisheggs_w_annotations](https://user-images.githubusercontent.com/8705477/86452376-80735f00-bd1c-11ea-8394-ba5f7a06fc6d.jpg)

Example of Larvae annotations:
![Larf 10_improved_w_annotations](https://user-images.githubusercontent.com/8705477/86452380-81a48c00-bd1c-11ea-9ba5-73ff08de2e16.jpg)

**Comment by cmspinto** [comment]
**Eggs**
Fields to provide with the annotation

1. Species
2. Development stage
3. Presence or absence of Oil globe
4. Segmented Yolk or not

Get to the species we need to provide the following optional settings
- Egg Diameter
- Oil globule diameter


**Larvae**
Fields to provide with the annotation
1. Species

Get to the species we need to provide the following optional settings
2. Total length
3. Standard length
4. Number of miotomes
5. Head length
6. Anal fin (yes or no)
7. Pelvic fin (yes or no)
8. Dorsal fin (yes or no)
9. Pylorus
10. Development stage

**Comment by KloppmannMH** [comment]
no. 9 should read: Number of myotomes between pylorus and **pelvic** fin

**Comment by KloppmannMH** [comment]
Eggs.
For egg identification, nos. 3. and 4. are also important and should be moved to the section below (to get to the species we need to provide....)
For Larvae, total and standard length (nos. 2 and 3) could be moved to the annotations section. However, lengths could also be provided with the data, which are given with each picture/sample

**Comment by CindyvanDamme** [comment]
Hi Matthias,

Yes you are right should be pelvic fin.

- for larvae only species is mandatory

- for eggs numbers 1-4 are mandatory

**Comment by KloppmannMH** [comment]
OK, those are mandatory fields. That's OK then

**Comment by JCoadDavies** [comment]
[Egg and larval events on SmartDots Aug2020.docx](https://github.com/ices-eg/SmartDots/files/5096045/Egg.and.larval.events.on.SmartDots.Aug2020.docx)

Meeting notes from today

**Comment by CindyvanDamme** [comment]
[Egg and larval events on SmartDots Aug2020_CvD.docx](https://github.com/ices-eg/SmartDots/files/5101793/Egg.and.larval.events.on.SmartDots.Aug2020_CvD.docx)

Comments on the meeting notes

**Comment by JCoadDavies** [comment]
[Egg and Larvae SmartDots Feedback 2020.xlsx](https://github.com/ices-eg/SmartDots/files/9233529/Egg.and.Larvae.SmartDots.Feedback.2020.xlsx)

**Comment by JCoadDavies** [comment]
[Egg and larval events on SmartDots Aug2020.docx](https://github.com/ices-eg/SmartDots/files/9233540/Egg.and.larval.events.on.SmartDots.Aug2020.docx)
